storage/connect/bsonudf.cpp

```diff
@@ -3571,14 +3571,14 @@ char *bson_item_merge(UDF_INIT *initid, UDF_ARGS *args, char *result,
 	if (!CheckMemory(g, initid, args, 2, false, false, true)) {
 		JTYP  type;
 		BJNX  bnx(g);
-		PBVAL jvp, top = NULL;
+		PBVAL jvp = NULL, top = NULL;
```

Comment:
All good to merge from me.

Reply:
Yep, looks good to me. What I may add is an assert !null where the value first gets used in the function to make it obvious of the assumption.